rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -30,7 +30,8 @@
     'ConsentPII': None,
     'DVEHRSharing': 'DVEHRSharing_AreYouInterested',
     'EHRConsentPII': 'EHRConsentPII_ConsentPermission',
-    'GROR': 'ResultsConsent_CheckDNA'
+    'GROR': 'ResultsConsent_CheckDNA',
+    'PrimaryConsentUpdate': 'Reconsent_ReviewConsentAgree'
 }
 
 # _consent_expired_question_map must contain every module ID from _consent_module_question_map.
```

Comment:
Comment suggests this list also needs to be updated to add PrimaryConsentUpdate?

Reply:
Thanks for noticing that. Updating it now.